--- a/torch/CMakeLists.txt	2026-02-01 09:49:20.971385420 +0200
+++ b/torch/CMakeLists.txt	2026-02-03 11:35:37.212261031 +0200
@@ -317,6 +317,18 @@
 
 
 add_library(torch_python SHARED ${TORCH_PYTHON_SRCS})
+# -----------------------------------------------------------------------------
+# Force pure-C compilation for Python 3.12 internal C sources
+# -----------------------------------------------------------------------------
+set_source_files_properties(
+    ${TORCH_SRC_DIR}/csrc/dynamo/cpython_defs.c
+    ${TORCH_SRC_DIR}/csrc/dynamo/eval_frame.c
+    ${TORCH_SRC_DIR}/csrc/dynamo/stackref_bridge.c
+    PROPERTIES
+        LANGUAGE C
+        COMPILE_OPTIONS "-std=gnu11"
+)
+# -----------------------------------------------------------------------------
 torch_compile_options(torch_python)  # see cmake/public/utils.cmake
 if(APPLE)
   target_compile_options(torch_python PRIVATE
