diff --git a/3rd/pybind11/pybind11.SlackBuild b/3rd/pybind11/pybind11.SlackBuild
index 8c2162a..f704670 100755
--- a/3rd/pybind11/pybind11.SlackBuild
+++ b/3rd/pybind11/pybind11.SlackBuild
@@ -26,7 +26,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=pybind11
-VERSION=${VERSION:-2.11.1}
+VERSION=${VERSION:-3.0.2}
 BUILD=${BUILD:-1}
 TAG=${TAG:-_SBo}
 PKGTYPE=${PKGTYPE:-tgz}
@@ -79,7 +79,7 @@ find -L . \
 
 mkdir build
 
-python3 setup.py build
+python3 -m build
 
 (
   cd build
@@ -93,7 +93,7 @@ python3 setup.py build
   make
 )
 
-python3 setup.py install --prefix=/usr --root="$PKG" --install-headers=/usr/include/pybind11 --skip-build
+python3 -m pip install dist/*.whl --root="$PKG" --prefix=/usr --no-deps
 
 (
   cd build
